### DISCUSSION TITLE
A2Z Channel 11

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#650] [Q&A]
**Channel Name:** A2Z
**Channel ID:** A2Z.ph
**Country:** Philippines
**Language:** Tagalog
**External Link (optional):** xxx
**Notes (optional):** Free to air channel in the Philippines.

**Comment by luke026** [comment]
Do u have of this url

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/ph.m3u#L2-L3
